pkg/virt-controller/services/vm.go

```diff
@@ -33,6 +33,7 @@ import (
 	corev1 "kubevirt.io/kubevirt/pkg/api/v1"
 	"kubevirt.io/kubevirt/pkg/logging"
 	"kubevirt.io/kubevirt/pkg/precond"
+	registrydisk "kubevirt.io/kubevirt/pkg/registry-disk"
```

Comment:
This is becoming a little to ugly, if we do storage specific initialization in the controller.